docs/language/learn-ql/ql-etudes/river-crossing-1.ql

```diff
@@ -0,0 +1,98 @@
+/**
+ * @name River crossing puzzle (version 1)
```

Comment:
Did you intend to include this file? It looks as if it might be an earlier version of `river-crossing.ql`

Reply:
I've included it in a few places. It introduces more simple definitions before using the "tidier" (but less intuitive) helper predicates and recursion. 
(I'll add a note to the top of the file to explain this!)